qutip/qobj.py

```diff
@@ -1794,9 +1821,15 @@ def groundstate(self, sparse=False, tol=0, maxiter=100000, safe=True):
         -------
         eigval : float
             Eigenvalue for the ground state of quantum operator.
-        eigvec : :class:`qutip.Qobj`
+        eigvec : :obj:`.Qobj`
             Eigenket for the ground state of quantum operator.
 
+        Raises
+        -------
+        WARNING : Ground state may be degenerate.
+            Use Q. :obj:`.eigenstates()`
+
+
```

Comment:
there's no need to mention warnings at all, but if you were to, they'd go in a "Warns" section, not "Raises".